fcrepo-export/src/test/java/org/fcrepo/exporter/ExporterTest.java

```diff
@@ -146,6 +152,18 @@ public void testExportContainer() throws Exception {
         Assert.assertTrue(exporter.wroteFile(new File("target/rdf/rest/1.jsonld")));
     }
 
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
```

Comment:
Ah, you need to examine the exception, I see. Yeah, I would actually prefer that you explicitly tangle out and assert about the exception in the test, because it's only one test. If there were a couple of different tests, I could see the savings in using the rule.